rabbus.go

```diff
@@ -237,24 +250,31 @@ func (r *rabbus) Listen(c ListenConfig) (chan ConsumerMessage, error) {
 }
 
 // Close attempt to close channel and connection.
-func (r *rabbus) Close() {
-	r.ch.Close()
-	r.conn.Close()
+func (ri *RabbusInterpreter) Close() (err error) {
+	if err = ri.ch.Close(); err != nil {
+		return err
```

Comment:
Shouldn't be just? https://play.golang.org/p/VKEF3WwA2v

```go
if err = ri.ch.Close(); err != nil {
     return
}
```

Reply:
Good catch!!